source-code/app/src/main/java/org/buildmlearn/toolkit/templates/ComprehensionAdapter.java

```diff
@@ -42,7 +44,7 @@
 
     private static final int TYPE_HEADER = 0;
     private static final int TYPE_ITEM = 1;
-    private final Context context;
+    private static  Context context;
```

Comment:
Using static field for Context variable will lead to memory leak. You can many article on memory leak through static fields in web. 
Can you come up with different approach for this?